tests/big_query/test_big_query.py

```diff
@@ -167,6 +177,16 @@ def __create_tables_list_responses():
              content('tests/json_samples/bigquery_table_list_page_last.json'))
         ])
 
+    @staticmethod
+    def __create_tables_list_404_responses():
```

Comment:
The name of this method is confusing. Maybe just call it '__create_dataset_not_found_response'